cms/models_test.py

```diff
@@ -50,6 +53,139 @@ def test_bootcamp_run_learning_resources():
         assert item.value.get("links") == "<p>links</p>"
 
 
+def test_three_column_image_text_section_under_runpage():
+    """
+    Three column image text section subpage should provide expected values.
+    """
+    bootcamp_run_page = BootcampRunPageFactory.create()
+    assert bootcamp_run_page.three_column_image_text_section is None
+    assert models.ThreeColumnImageTextSection.can_create_at(bootcamp_run_page)
+    three_column_image_text_page = ThreeColumnImageTextSectionFactory.create(
```

Comment:
this should be section rather than being named as page
please rename it to `three_column_image_text_section` and its usages
in order to maintain consistency with the rest of the cms app.